tests/integration.rs

```diff
@@ -299,6 +298,104 @@ fn tool_fail_if_thumnail_missing() -> Result<(), Box<dyn Error>> {
         .arg("-f")
         .assert()
         .failure()
-        .stderr(predicate::str::contains("resource not found"));
+        .stderr(str::contains("resource not found"));
+    Ok(())
+}
+
+#[test]
+fn test_succeed_using_example_signer() -> Result<(), Box<dyn Error>> {
+    let output = temp_path("./output_external.jpg");
+
+    let mut successful_process = PathBuf::from(env!("CARGO_MANIFEST_DIR"));
```

Comment:
It took me a while to figure out why we are calling something in target/debug here. It may help to have a comment explaining that we are running the compiled binary executables from this package here. It's nice that this works!

Reply:
Resolved here: https://github.com/contentauth/c2patool/pull/169/commits/af21581b597ee4400e017530d5ec4a14258b2bdc